app/screens/settings-screen/username-screen.tsx

```diff
@@ -34,14 +36,14 @@ type Props = {
   navigation: Record<string, any>
 }
 
-export const UsernameScreen = ({ navigation }: Props) => {
-  const [loading, setLoading] = React.useState(false)
-  const [input, setInput] = React.useState("")
-  const [message, setMessage] = React.useState("")
-  const [messageIsError, setMessageIsError] = React.useState(false)
+export const UsernameScreen: ScreenType = ({ navigation }: Props) => {
+  const [loading, setLoading] = useState(false)
+  const [input, setInput] = useState("")
+  const [message, setMessage] = useState("")
+  const [messageIsError, setMessageIsError] = useState(false)
 
-  // we don't show a error message before the user had a change to add 3 caracters
-  const [init, setInit] = React.useState(true)
+  // we don't show a error message before the user had a change to add 3 characters
+  const [init, setInit] = useState(true)
```

Comment:
Since you're also cleaning up in this PR. Can we rename this to something more explicit like `shouldShowErrorMessage` or something like that?